Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -675,6 +682,78 @@ void GameList::OpenSaveFolder()
   QDesktopServices::openUrl(url);
 }
 
+void GameList::OpenGCSaveFolder()
+{
+  const auto game = GetSelectedGame();
+  if (!game)
+    return;
+
+  std::string region;
+
+  switch (game->GetRegion())
```

Comment:
Perhaps this could be made part of the `GameFile` interface, like `GetRegionString`/`GetRegionAsString` or something.